sample_apps/README.md

```diff
@@ -61,12 +61,11 @@ You can use this script to Build and run example programs.
 
 **Usage**:
 ```shell
-run_example.sh [-h | --help] < sample-app-name > [ setup | run_test | --list ]
+run_example.sh [-h | --help | --list] <sample-app-name> [ setup | run_test ]
```

Comment:
Made the interface a bit more consistent with `--help` and `--list` coming up-ahead.